### PR TITLE
[Early Mirror] Server announcements are no longer ock ick

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

tgstation 80741/80849

## Proof Of Testing

image

### CODE REVIEW

**Comment by StrangeWeirdKitten** [comment]
The changelog will automatically come down, and should be removed in the future.

Also I'm going to wait for https://github.com/tgstation/tgstation/pull/80849 to be merged. Early pulls need to have no code changes for obvious reasons. 

But it can still be test merged. 👍

**Comment by lessthnthree** [comment]
Given the latest commit was just documentation, yeah I'll make it identical by the time it's fully merged.

File: code/__HELPERS/announcements.dm [hunk 1]
```diff
@@ -16,20 +16,20 @@
...
 	text,
 	title = "",
 	players,
 	play_sound = TRUE,
-	sound_override = 'sound/ai/default/attention.ogg',
+	sound_override = 'sound/misc/bloop.ogg',
 	sender_override = "Server Admin Announcement",
 	encode_title = TRUE,
-	encode_text = TRUE,
+	encode_text = FALSE,
```
Comment (Majkl-J) [inline code/__HELPERS/announcements.dm:32]: Look how they massacred my boy

Comment (Majkl-J) [inline code/__HELPERS/announcements.dm:32]: Look how they massacred my boy

Comment (lessthnthree) [reply code/__HELPERS/announcements.dm:32]: this change allows HTML in announcements again. the encode_text arg strips HTML in the announcement

**Pull request closed without merging**